product_docs/docs/biganimal/release/overview/03_security/01_securing_your_cluster.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: "Customizing policy definitions"
```

Comment:
To be clear, what is the title of this page/section? I see the document title still refers to cluster - but this page doesn't have anything to do with "clusters" as defined elsewhere in the docs, or really even "securing".

Reply:
merged the pages and renamed the file.